pkg/cli/tool_base.go

```diff
@@ -115,7 +115,8 @@ func (c *ToolBaseCommand) setup(ctx context.Context, args []string) (*v1alpha1.T
 		h = c.testHandler
 	} else {
 		opts := []handler.ToolHandlerOption{handler.WithStderr(c.Stderr())}
-		if c.flagDebug {
+		if c.flagVerbose {
+			printHeader(c.Stdout(), "Commands Output")
```

Comment:
Note that since we are using a testHander interface here, cannot really unit test the handler opts, to show how it looks with echo commands:
<img width="477" alt="Screenshot 2023-07-28 at 3 31 41 PM" src="https://github.com/abcxyz/access-on-demand/assets/51539171/27f6caf3-e2b1-40fc-943a-825463f7993a">

Reply:
Can we output a command followed by its output?

Reply:
updated to below:
<img width="465" alt="Screenshot 2023-07-28 at 4 06 51 PM" src="https://github.com/abcxyz/access-on-demand/assets/51539171/a66239ed-20f7-42f5-abfd-e1f9bb5e1e94">

Reply:
how about
<img width="477" alt="Screenshot 2023-07-28 at 5 53 30 PM" src="https://github.com/abcxyz/access-on-demand/assets/51539171/ecced160-e39a-48c0-a365-56474a01bc78">